installer/Migration/V5_0_0/Migration.php

```diff
@@ -336,6 +337,20 @@ private function configureMenuItems(string $configName, string $parentMenuTitle)
             ->executeQuery();
     }
 
+
+    /**
+     * @return void
+     */
+    private function addValidColumnToRequestResetPassword(): void
```

Comment:
this is correct. but have to create a new `Migration.php` file under
`installer/Migration/V5_1_0`